app/coffee/PersistenceManager.coffee

```diff
@@ -28,6 +28,7 @@ module.exports = PersistenceManager =
 				sendImmediately: true
 			jar: false
 			timeout: MAX_HTTP_REQUEST_LENGTH
+			maxAttempts: 2  # for requestretry
```

Comment:
Couple of points I noticed, keeping in mind we only have 30 seconds in the lock:-
-  the default delay in requestretry is 5 seconds, too long for us. Suggest turning it down to 100ms.
-  since we have a short timeout on the request of 5 seconds  we should avoid doubling it and only retry on HTTP errors (I think the original problem was 500s right?)

```
+			retryDelay: 100 # the default delay is 5000ms
+			retryStrategy: request.RetryStrategies.HTTPError # don't retry on timeouts
```